### PR TITLE
Update BUILD.txt

### PR DESCRIPTION
**Pull request by brxxm**
Changed some errors/omissions I found while building an image for R 2.4.0

### CODE REVIEW

**Comment by houz** [comment]
I can't comment on the changes you did, but shouldn't this go to the master branch?

**Comment by brxxm** [comment]
Yes. Apparently I did something wrong. I do not know how to correct this.

File: packaging/macosx/BUILD.txt [hunk 1]
```diff
@@ -25,12 +25,18 @@ How to make disk image with darktable application bundle (64 bit Intel only):
...
+    Comment out 'macosx_deployment_target 10.7' that you added in file '/opt/local/etc/macports/macports.conf' (refer to line 6) then
+    $ port install llvm-5.0
+    $ port install llvm_select
+    Remove the comment # before 'macosx_deployment_target 10.7' in the file '/opt/local/etc/macports/macports.conf' and proceed:
      $ sudo port install gcc6
-    (llvm port, which gets installed as dependency, won't compile with macosx_deployment_target set, so comment it out when llvm is built)
     Clone darktable git repository (in this example into ~/src):
      $ mkdir ~/src
      $ cd ~/src
      $ git clone git://github.com/darktable-org/darktable.git
```
Comment (flannelhead) [inline packaging/macosx/BUILD.txt]: Not a big deal, but could pass `--recursive` to `git clone` here - this would make the next two steps unnecessary.

**Comment by parafin** [comment]
Merged manually with 7eb11afe0eae3cb023601b8b7255358cfe99b996
Thanx @brxxm and @flannelhead!
Original paths to portfiles work for me, so I chose more generic approach.
As for llvm clarification - I think current verbosity level of this document is enough, it's intended for developers who understand what happens when these commands are run. Blindly following the instructions won't achieve much, this is not a script for a build machine and never was intended as such.

**Pull request closed without merging**